HARSHAL/javascript/assignment10/assignment10Q5.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>que5</title>
+    </head>
+    <body>
+        <script src="assignment10Q5.js"></script>
```

Comment:
js file is missing. No need to add js files u can write the script inside the same html file